assets/app/view/game/round/auction.rb

```diff
@@ -109,6 +110,28 @@ def render_show_button
           h(:button, props, "#{hidden? ? 'Show' : 'Hide'} #{@step.visible? ? 'Player' : 'Companies'}")
         end
 
+        def render_confirm_selections
+          return nil if hidden? && !@step.visible?
+          return nil unless @current_actions.include?('select_multiple_companies')
+
+          mutiple_companies = lambda do
+            hide!
+            process_action(Engine::Action::SelectMultipleCompanies.new(
+              @current_entity,
+              companies: @step.selected_companies
+            ))
+          end
+          select_button = [h(:button, { on: { click: mutiple_companies } }, 'Confirm Selections')]
+
+          children = []
+          if @step.respond_to?(:selection_note)
+            note = @step.selection_note.map { |text_block| h(:p, text_block) }
+            children << h(:div, note)
+          end
```

Comment:
Again, you don't need to include the `respond_to?` check.

Reply:
The `respond_to?` check is useful here that a `selection_note` can be an optional feature.

Reply:
It's not an optional feature at the moment, as 18India will be the only game that has the multiple selection and it uses the selection note. If another game reuses multiple selection and doesn't want a note shown it could be made optional at that time. (And if it were to be optional I'd prefer that this would be done by `selection_note` returning nil or an empty array).

Reply:
That makes sense.  I do have a tendency to overthink future possibilities at times.  

I have removed the `respond_to?` check and simplified the note section.